cli.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"strconv"
+)
+
+var batchmode = flag.Bool("batch", false, "Whether to run in batch mode (requires -decks and -out)")
+var deckpath = flag.String("decks", "", "(batch mode) A directory with subdirectories containing deck files")
+var outpath = flag.String("out", "", "(batch mode) Output directory for aggregated decks; will be created if it doesn't exist")
+var servermode = flag.Bool("web", false, "Whether to run in webserver mode")
```

Comment:
Can servermode default to true? I'd like the default behavior of the no-args invocation to be consistent with the behavior before batch mode was added. Looks like you've made an effort to do that with the other flags?

Reply:
It does default to true in that, if not specified, servermode is chosen. The `false` here is needed to determine whether the flag is given on the command line.

Reply:
Gotcha, I misread.